Vizzuality/half-earth-v3
# Feature/sidebar styling

## Sidebar style updates
### Description
- Update search subtitle
- Add sidebar hover
- Update radio gradient
- Add mocked text cards
- Change existing scrollbars and add missing ones
- Fix width to show scrollbar on ranking
- Fix species modal open

Missing: Sidebar thumbnails

### Designs
https://projects.invisionapp.com/d/main?origin=v7#/console/21108730/445055730/preview?scrollOffset=0
### Testing instructions

- Use sidebar hover and check updated design
- Try and find scroll bars
- Test species modal on the country page

### Feature relevant tickets
https://half-earth-map.atlassian.net/browse/HE-294
https://half-earth-map.atlassian.net/browse/HE-251
https://half-earth-map.atlassian.net/browse/HE-261
